### ISSUE TITLE
Gunlord (HB)

### ISSUE THREAD

**Issue by delon5** [Rinnegatamante/flycast-vita-compatibility-list#189]
game img: cdi

Gunlord targets 60fps, however when in game it’s pretty much a solid 30 for the majority of the time. Which makes your movements feel like slow motion.

![A651A896-A2E5-41AB-8A81-D9FABA714D80](https://user-images.githubusercontent.com/25923990/187124198-91dc67b1-c586-4d2b-98d9-5f834ddd35b4.jpeg)

![D41F8FAE-B8BD-4036-8BCB-8DA934015B71](https://user-images.githubusercontent.com/25923990/187124219-c02bd7c4-ef98-4a86-8f62-00ae08513082.jpeg)

![D616C1A5-DC6A-467F-84CB-CE4531675346](https://user-images.githubusercontent.com/25923990/187124238-234c2581-fbbd-4061-b011-dd5283253d67.jpeg)

this would most likely benifit from a 30fps patch.

I would rate it ingame+.

GENERAL
pal
tv composite

VIDEO
Frame skipping - max
Per Strip
Shadows - yes
fog - yes
mipmaps - yes
use simple shaders - yes
delay frame swaping - yes

ADVANCED
idle skip
use neon simd
self-modifying code checks - off
multi-threaded emulation

**Comment by Rinnegatamante** [comment]
Try to reduce the downclock and to change float ops hack.

**Comment by delon5** [comment]
1.000 Downclock is best for gameplay more consistent fps less slow motion. Terrible audio

5.000 Downclock is great audio with slow motion. 

I tried to find a middle ground with 2.000 downclock. And I am unsure if float hack makes that much of a difference.

**Comment by DieterB-22** [comment]
With these settings the game is very good playable.
Clear Sound with some dips.
50~60Fps with some dips when too much on screen.

NTSC
USA
TV Composite

VIDEO following options must be enabled
Per Triangle  
Shadows
Fog
Use Mimaps 
Use simple shaders
VSync
Delay Frame Swapping 


ADVANCED following options must be enabled
Idle Skip
Use Neon SIMD
Enable UPnP?!?
Downclock 1500 
Multi Threaded Emulation
Force Windows CE should be activated and can't be switched off.

Tag:playable

**Comment by delon5** [comment]
These are the same settings I tested and posted multiple options for preferred results. Game is still frustrating to play with a lot of slowdown when fighting enemy’s. Game is still in game+. Not enjoyable to play.

**Comment by DieterB-22** [comment]
Gunlord pretty good playable.
https://youtu.be/evEX5fPb6Ck

**Comment by delon5** [comment]
> Gunlord pretty good playable.
> https://youtu.be/evEX5fPb6Ck

Yeah we have the same performance, I don’t think playing a game a 60%-75% movement speed is an enjoyable way to play this game. If you haven’t played it on Dreamcast which is highly enjoyable here is what it should look like https://youtu.be/fXiDHriOJUw. If it was like this 80% of the time while fighting enemies I would rate it playable. But it isn’t and never reaches full speed in fights not enjoyable there for ingame+.
